src/oaklib/cli.py

```diff
@@ -1922,6 +1922,27 @@ def descendants(terms, predicates, display: str, output_type: str, output: TextI
     show_default=True,
     help="For formats that export only IS_A by default, this will include all possible predicates",
 )
+@click.option(
+    "--use-curies-native-concepts/--no-use-curies-native-concepts",
```

Comment:
I could probably use some improvement on the descriptions for these CLI options, but for now, here's what I've got:

`--use-curies-native-concepts`
> For FHIR. FHIR conventionally uses codes for references to concepts that are native to a given CodeSystem. With this option, references will be CURIEs instead.

`--use-curies-foreign-concepts`
> For FHIR. Typical FHIR CodeSystems do not contain any concepts that are not native to that CodeSystem. In cases where they do appear, this converter defaults to URIs for references, unless this flag is present, in which case the converter will attempt to construct CURIEs.

`--native-uri-stems`
> For FHIR. A comma-separated list of URI stems that will be used to determine whether a concept is native to the CodeSystem. For example, for OMIM, the following URI stems are native: https://omim.org/entry/,https://omim.org/phenotypicSeries/PS

Reply:
unfortunately the CLI will blow up if we expose all possible configurations of all possible systems

What I would recommend here is that the CLI accepts as argument a generic configuration file or object, and specific converters can then define their own fields here

Note the DumperInterface already looks like this:

```
    def dump(
        self, path: str = None, syntax: str = None, enforce_canonical_ordering=False, **kwargs
    ):
        """
        Exports current contents.

        :param path: Path to file to write to. If None, then write to stdout.
        :param syntax: Syntax to use. If None, then use the default syntax.
        :param enforce_canonical_ordering:
        :param kwargs: Additional arguments to pass to the dumper
        :return:

...
...
        converter.dump(ogdoc, target=path, **kwargs)
```

so the CLI could be made to accept a yaml file that is just flattened into `**kwargs`

Reply:
Thanks for the feedback, Chris. I am just unsure of how you want me to implement, I can think of several options. See [Slack](https://obo-communitygroup.slack.com/archives/D046Y7BPEJH/p1680199597627229).